pytom/alignment/localOptimization.py

```diff
@@ -381,33 +381,3 @@ def design_fsc_filter(fsc, fildim=None, fsc_criterion=0.143):
         else:
             fil[ii] = fsc_fil[ii]
     return fil
-
-
-def alignVolumes(particle, reference, referenceWeighting, wedgeInfo, iniRot, iniTrans,
-                 scoreObject=0, mask=None, preprocessing=None, progressBar=False, binning=1,
-                 bestPeak=None, verbose=False):
-    """
-    Analog to bestAlignment function for local optimization
-    @param particle: A particle
-    @type particle: L{pytom_volume.vol}
-    @param reference: A reference
-    @type reference: L{pytom_volume.vol}
-    @param referenceWeighting: Fourier weighting of the reference (sum of wedges for instance)
-    @type referenceWeighting: L{pytom.basic.structures.vol}
-    @param wedgeInfo: What does the wedge look alike?
-    @type wedgeInfo: L{pytom.basic.structures.Wedge}
-    @param iniRot: initial guess for rotation
-    @param iniTrans: initial guess for translation
-    @param scoreObject:
-    @type scoreObject: L{pytom.score.score.Score}
-    @param mask: real-space mask for correlation function
-    @type mask: L{pytom.basic.structures.Particle}
-    @param preprocessing: Class storing preprocessing of particle and reference such as bandpass
-    @type preprocessing: L{pytom.alignment.preprocessing.Preprocessing}
-    @param progressBar: Display progress bar of alignment. False by default.
-    @param binning: Is binning applied (currently not properly functioning)
-    @param bestPeak: Initialise best peak with old values.
-    @param verbose: Print out infos. Writes CC volume to disk!!! Default is False
-    @return: Returns the best rotation for particle and the corresponding scoring result.
-
-    """
```

Comment:
Was this dead code?

Reply:
Yep, it was a function header without any code in it.